perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/UsersManagerMethod.java

```diff
@@ -705,6 +705,54 @@ public UserExtSource call(ApiCaller ac, Deserializer parms) throws PerunExceptio
 		}
 	},
 
+	/*#
+	 * Return userExtSource for specific attribute definition (specified by id) and unique value.
+	 * If not found, throw and exception.
+	 *
+	 * It looks for exactly one value of the specific attribute type:
+	 * - Integer -> exactly match
+	 * - String -> exactly match
+	 * - Map -> exactly match of "key=value"
+	 * - ArrayList -> exactly match of one of the value
+	 *
+	 * @param attributeId attribute id used for founding attribute definition which has to exists, be unique and in userExtSource namespace
+	 * @param attributeValue value used for searching
+	 * @return userExtSource found by attribute id and it's unique value
```

Comment:
Also, you should specify the type of the returned object.

Reply:
Done